sklearn/utils/_indexing.py

```diff
@@ -451,6 +459,10 @@ def resample(*arrays, replace=True, n_samples=None, random_state=None, stratify=
         If not None, data is split in a stratified fashion, using this as
         the class labels.
 
+    sample_weight : sequence of array-like of shape (n_samples,) or \
```

Comment:
I don't think we want to support a sequence of sample weight arrays. At least the current implementation doesn't. I think it's reasonable for now to impose the same sample weight array for all arrays. Thus it should be just `array-like of shape (n_samples,) or None, default=None`.